### PR TITLE
feat: add qdwh() function to jax.lax frontend

### LINKED ISSUE #25633: qdwh

### PR DESCRIPTION
**Pull request by rukiyearican**
# PR Description

added qdwh and qdwh test

## Related Issue

Closes #25633

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by rukiyearican** [comment]
hello @RickSanchezStoic 

this is my first pull request, therefore i'm not sure i did all of correctly. i would be very happy if you can comment. i receive some assertion errors in local testing, im still trying to figure it out, that's why im not sure

**Comment by rukiyearican** [comment]
hello @hmahmood24 

this is my first pull request, therefore i'm not sure i did all of correctly. i would be very happy if you can comment. i receive some assertion errors in local testing, im still trying to figure it out, that's why im not sure

File: ivy/functional/frontends/jax/lax/linalg.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import ivy
 from ivy.functional.frontends.jax.func_wrapper import to_ivy_arrays_and_back
 from ivy.func_wrapper import with_unsupported_dtypes
+import numpy as np_frontend
```
Comment (hmahmood24) [inline ivy/functional/frontends/jax/lax/linalg.py:4]: We're not supposed to use native packages like `jax`, `numpy`, `torch` etc. in the frontends. Instead, we only implement the frontend functions using either `ivy` functions or functions from the same frontend e.g. `jax` frontend

File: ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py [hunk 2]
```diff
@@ -119,6 +120,83 @@ def test_jax_eigh(
...
+        # Check convergence based on epsilon
+        x_norm = np.linalg.norm(x)
+        y = np.linalg.norm(h) * (4 * eps) ** (1 / 3)
+        if x_norm < y:
+            assert is_converged
+
+    if max_iterations:
+        # Check if the maximum number of iterations is reached
+        if num_iters >= max_iterations:
+            assert not is_converged
```
Comment (hmahmood24) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_linalg.py:197]: Please look at all the other tests in this file and all the other test files to get a feel of how to write the correct tests. You don't need to add assert statements manually here. Neither do you need to import the `qdwh` function from the jax frontend. The `helpers.test_frontend_function` already handles this internally by checking if the output of the frontend function and the actualy `jax.lax.linalg.qdwh` match or not.

**Review by hmahmood24 — CHANGES_REQUESTED** [review]
Hey @rukiyearican, thanks for the PR. I left a few comments for you. It would also help to go through our docs on unifyai.com to get a feel of how to write frontend functions and frontend function tests. Thanks!

**Comment by hmahmood24** [comment]
Closing this PR since it has been stale for more than 2 weeks now.

**Pull request closed without merging**